CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ option(BUILD_VINEYARD_CLIENT "Build vineyard's client" ON)
 option(BUILD_VINEYARD_CLIENT_VERBOSE "Build vineyard's client with the most verbose logs" OFF)
 option(BUILD_VINEYARD_PYTHON_BINDINGS "Build vineyard's python bindings" ON)
 option(BUILD_VINEYARD_PYPI_PACKAGES "Build vineyard's python bindings" OFF)
+option(BUILD_VINEYARD_LLM_KV_CACHE_PYTHON_BINDINGS "Build vineyard llm kv cache's python bindings" ON)
```

Comment:
No need for this extra option.

`if (BUILD_VINEYARD_PYTHON_BINDINGS AND BUILD_VINEYARD_LLM_CACHE)` would be enough.

Reply:
Done.